steps/index.lab.md

```diff
@@ -54,93 +53,154 @@ $ git clone https://github.com/firebase/codelab-friendlychat-android
+}
 
-1. From the overview screen of your new project, click the Android icon to launch the setup workflow.
-2. Enter the codelab's package name: `com.google.firebase.codelab.friendlychat`
-3. Enter the SHA1 of your signing keystore. If you are using the standard debug keystore, use the command below to find the SHA1 hash:
+dependencies {
+    // ...
 
-```console
-keytool -alias androiddebugkey -keystore ~/.android/debug.keystore -list -v -storepass android
+    // Google Sign In SDK
+    implementation 'com.google.android.gms:play-services-auth:19.0.0'
+
+    // Firebase SDK
+    implementation platform('com.google.firebase:firebase-bom:26.4.0')
+    implementation 'com.google.firebase:firebase-database'
+    implementation 'com.google.firebase:firebase-storage'
+    implementation 'com.google.firebase:firebase-auth'
+
+    // Firebase UI Library
+    implementation 'com.firebaseui:firebase-ui-database:7.1.1'
+}
+
+// Apply the 'google-services' plugin
+apply plugin: 'com.google.gms.google-services'
 ```
 
-> aside positive
-> 
-> **Note**: Your debug keystore, usually called "debug.keystore", is typically located at 
-> 
-> $HOME/.android/debug.keystore.  If this file does not exist, Android Studio will create it for you the first time you run any app.
-> 
-> See  [here](https://developers.google.com/android/guides/client-auth) for more detail on finding SHA1.
 
-#### **Add google-services.json file to your app**
+## Create Firebase project
+Duration: 06:00
 
-After adding the package name and SHA1 and selecting Register**, Click Download google-services.json** to obtain your Firebase Android config file then copy the google-services.json file into the *`app`* directory in your project. After the file is downloaded you can **Skip** the next steps shown in the console (they've already been done for you in the build-android-start project).
+In this step you will create a Firebase project to use during this codelab and add the project configuration to your ap.
 
-#### **Add google-services plugin to your app**
+### Create a new project
 
-The google-services plugin uses the google-services.json file to configure your application to use Firebase. The following line should already be added to the end of the build.gradle file in the *`app`* directory of your project (check to confirm):
+1. In your browser go to the  [Firebase console](https://console.firebase.google.com).
+2. Select **Add project**.
+3. Select or enter a project name, you can use any name you want.
+4. You will not need Google Analytics for this project, so you can disable it when asked.
+5. Click **Create Project** and when your project is ready click **Continue**
 
-```
-apply plugin: 'com.google.gms.google-services'
+### Add Firebase to your app
+
+Before you begin this step, get the SHA1 hash of your app: Run the command below in the project directory to determine the SHA1 of your debug key:
+
+```console
+./gradlew signingReport
+
+Store: /Users/username/.android/debug.keystore
```

Comment:
Is there some way to call out "username" as a placeholder here? Maybe:

`/Users/<username>/.android/debug.keystore`